scripts/build.sh

```diff
@@ -79,6 +86,13 @@ KERNELVERSION="${VERSION}" \
         "${MODEL_DIR}/coreboot.config" \
         "${COREBOOT}"
 
+# Rebuild EC firmware for System76 EC models
+if [ -n "${EC_MODEL[$MODEL]}" ]
+then
+    make -B -C ec -B BOARD=system76/${EC_MODEL[$MODEL]} VERSION=${VERSION}
```

Comment:
I recommend creating a script _build/ec.sh, that expects an argument for the model config, and an argument indicating where the ec ROM goes.

Script would be called like this:
```
env VERSION="${VERSION}" \
    ./scripts/_build/ec.sh \
    "${MODEL_DIR}/ec.config" \
    "${BUILD}/ec.rom"
```

That script would do something like this (see other scripts for verifying arguments)"
```
source "$1"
make -C ec clean
make -C ec -j "$(nproc)"
cp "ec/build/${BOARD}/${VERSION}/ec.rom $2"
```